### PR TITLE
PWGGA/GammaConv added nonlin correction for LHC13b4, JJ MC LHC13bcdef

### PR DESCRIPTION
**Pull request by mikesas**

### CODE REVIEW

File: PWGGA/GammaConv/AliCaloPhotonCuts.cxx [hunk 1]
```diff
@@ -5514,6 +5514,10 @@ void AliCaloPhotonCuts::ApplyNonLinearity(AliVCluster* cluster, Int_t isMC)
             energy /= FunctionNL_kSDM(energy, 0.987513, -4.34641, -0.522125) ;
             energy /= 0.9935;
           }
+        if( fCurrentMC==kPPb5T13P4JJ ){
```
Comment (dmuhlhei) [inline PWGGA/GammaConv/AliCaloPhotonCuts.cxx]: shouldn't you use 'else if'

File: PWGGA/GammaConv/AliCaloPhotonCuts.cxx [hunk 2]
```diff
@@ -5551,6 +5555,10 @@ void AliCaloPhotonCuts::ApplyNonLinearity(AliVCluster* cluster, Int_t isMC)
             energy /= (FunctionNL_kSDM(energy, 0.987611, -4.14227, -0.282541) * 1.0036264536 );
             energy /= 0.9935;
           }
+        if( fCurrentMC==kPPb5T13P4JJ ) {
```
Comment (dmuhlhei) [inline PWGGA/GammaConv/AliCaloPhotonCuts.cxx]: same as above

File: PWGGA/GammaConv/AliCaloPhotonCuts.h [hunk 3]
```diff
@@ -130,6 +130,7 @@ class AliCaloPhotonCuts : public AliAnalysisCuts {
       kPP8T12P2JJ,
       // pPb 5 TeV 2013
       kPPb5T13P2DPMJet,
+      kPPb5T13P4JJ,
```
Comment (dmuhlhei) [inline PWGGA/GammaConv/AliCaloPhotonCuts.h]: classdef update

File: PWGGA/GammaConv/AliCaloPhotonCuts.cxx (hunk 1 above)
Comment (mikesas) [reply PWGGA/GammaConv/AliCaloPhotonCuts.cxx]: Certainly... this was stupid of me.

**Comment by dmuhlhei** [comment]
+1

**Pull request merged**